po/POTFILES.in

```diff
@@ -50,6 +51,7 @@ plugins/xrdb/msd-xrdb-manager.c
 [type: gettext/ini]plugins/xrdb/xrdb.mate-settings-plugin.in
 plugins/xsettings/msd-xsettings-manager.c
 [type: gettext/ini]plugins/xsettings/xsettings.mate-settings-plugin.in
+[type: gettext/ini]plugins/rfkill/rfkill.gnome-settings-plugin.in
```

Comment:
Why rfkill.gnome-settings-plugin.in ?
Isn't rfkill.mate-settings-plugin.in the better file name?